src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenLocalFunctionTests.cs

```diff
@@ -3446,6 +3446,51 @@ Action<int> L5(int x)
         }
 
```

Comment:
I think we should have a test for calling a non-generic local function with a dynamic param in a generic type (which should succeed).